src/trackyr/notification_agents/routes.py

```diff
@@ -84,7 +84,7 @@ def test_notification_agent():
     form = NotificationAgentForm()
     json = request.json
     webhook_url = json['webhook']
-    if json['username']:
+    if not json['username']:
```

Comment:
Wouldn't this set username to nothing?

Reply:
Yes, sorry should be fixed now. I tested it and it does set the username appropriately.